src/backend/opencl/platform.cpp

```diff
@@ -22,8 +22,6 @@
 
 #endif
 
-#include <cl.hpp>
```

Comment:
:) That is the third task, i just made sure it is working as it is with old header using the external project setup. What @pavanky suggested on the issue is that functions/functors have become more terse. I have no idea what are the new things, so still have to see what this new header does. Then only i can change it to cl2.hpp.